pycbc/results/pygrb_plotting_utils.py

```diff
@@ -70,9 +54,109 @@ def contour_plotter(axis, snr_vals, contours, colors, vert_spike=False):
 
 
 #
-# Used (also) in executables
+# Functions used in executables
 #
 
+# =============================================================================
+# Plot trigger time and offsource extent over segments
+# Copyright (C) 2015 Andrew R. Williamson
```

Comment:
Don't think we need this copyright/comment (copyrights can go at the top, if needed) .... I also think the original version of this plot is courtesy of Alex Dietz (it exists even in the pre-coh_PTF GRB search results).)

Reply:
I had copied that function from a file with Andrew in the Copyright string on the top, so I thought I would keep track of that in this new file.  I now added `courtesy of Alex Dietz` instead.